pkg/reconciler/knativeeventing/controller.go

```diff
@@ -59,7 +59,12 @@ func NewController(
 
 	koDataDir := os.Getenv("KO_DATA_PATH")
 
-	cfg, err := clientcmd.BuildConfigFromFlags(*MasterURL, *Kubeconfig)
+	KubeconfigToUse := *Kubeconfig
```

Comment:
What about we remove all the changes, but replace
```
Kubeconfig = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
```

with 

```
Kubeconfig = flag.String("kubeconfig", os.Getenv("KUBECONFIG"), "Path to a kubeconfig. Only required if out-of-cluster.") along Line 42 in  pkg/reconciler/knativeeventing/controller.go?
```????

Reply:
Good point :D